corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -643,12 +649,23 @@ def get_doc(self, doc_id):
     def get_docs(self, from_db, startkey, last_doc_pk=None, limit=500):
         server_modified_on_since = startkey or datetime.min
         last_id = last_doc_pk or -1
-        results = CommCareCaseSQL.objects.raw(
-            'SELECT * FROM get_all_cases_modified_since(%s, %s, %s)',
-            [server_modified_on_since, last_id, limit],
-            using=from_db
+        domain_clause = "case_table.domain = %s AND" if self.domain else ""
+        values = [False, server_modified_on_since, last_id, limit]
+        if self.domain:
+            values = [self.domain] + values
+
+        # using raw query to avoid having to expand the tuple comparison
+        results = CommCareCaseSQL.objects.using(from_db).raw(
+            """SELECT * FROM {table} as case_table
+            WHERE {domain_clause}
+            deleted = %s AND
+            (case_table.server_modified_on, case_table.id) > (%s, %s)
+            ORDER BY case_table.server_modified_on, case_table.id
+            LIMIT %s;""".format(
+                table=CommCareCaseSQL._meta.db_table, domain_clause=domain_clause
```

Comment:
Is there a way I can name my domain `; DROP ALL TABLES;` or something similar where this might become an issue?